proxy/src/coco/source.rs

```diff
@@ -237,6 +242,18 @@ impl TryFrom<Revision> for Rev {
     }
 }
 
+/// Bundled response to retrieve both branches and tags for a user repo.
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct UserRevisions {
+    /// Owner of the repo.
+    pub(crate) identity: identity::Identity,
+    /// List of [`git::Branch`].
+    pub(crate) branches: Vec<Branch>,
```

Comment:
Should branches be `NonEmpty`? As far as I know you must have a commit and therefore a branch for the application to display a repo.

Reply:
I realize this wasn't added in this pr, so feel free to ignore :p

Reply:
_ignoring_ :joy: 

Surf doesn't return `NonEmpty`. So maybe you can create a ticket there :eyes:

Reply:
https://github.com/radicle-dev/radicle-surf/issues/151